src/spetlr/utils/CleanupTestDatabases.py

```diff
@@ -0,0 +1,34 @@
+import time
+
+from spetlr.config_master import Configurator
+from spetlr.exceptions import OnlyUseInSpetlrDebugMode
+from spetlr.spark import Spark
+
+_time_spent = 0
+
+
+def CleanupTestDatabases():
+    """
+    This function can be applied for removing test databases
+    in the Databricks Environment.
+    """
+
+    c = Configurator()
+    if not c.is_debug():
+        raise OnlyUseInSpetlrDebugMode()
+
+    start = time.time()
```

Comment:
let's remove this time spent printout. It was a quick debug tool that I once introduced.

Reply:
Done